### PR TITLE
[retrospective #17] Retrospective outcomes

### PR DESCRIPTION
**Pull request by arkhi**
## Summary of changes

- Headers are consistent with default Github templates.
- [multi-line] Rules to align operators at the start of the line.
- [templates] KISS for templates.

/fyi: @imlazyone & @tjoelsson: Please review.

/cc: @juwai/all: Give your opinion if you disagree with something.

### CODE REVIEW

File: language-php.md [hunk 1]
```diff
@@ -1,9 +1,19 @@
...
+As mentioned in _[php the right way][1]_, PHP-FIG, the PHP Framework Interop Group, announced PSR.
 
 We decided to follow [PSR-2].
 
 [1]: http://www.phptherightway.com/
 [PSR-2]: https://github.com/php-fig/fig-standards/blob/master/accepted/PSR-2-coding-style-guide.md
+
+## Operators
+
+Starting from PHP 5.3, you can use `?!` to [simplify ternary operations](https://php.net/manual/en/language.operators.comparison.php#language.operators.comparison.ternary). The two following statements are the same:
```
Comment (mark-juwai) [inline language-php.md]: `?:` ?

Comment (arkhi) [inline language-php.md]: Oh boy… Yes. :)

**Comment by mark-juwai** [comment]
👍 

Thanks for contributing to the documentation. :)

**Comment by tjoelsson** [comment]
👍

**Comment by arkhi** [comment]
Thanks guys. Merging…

**Pull request merged**